test/parallel/test-diagnostics-channel-module-require.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const dc = require('diagnostics_channel');
+
+const trace = dc.tracingChannel('module.require');
+const events = [];
+let lastEvent;
+
+function track(name) {
+  return (event) => {
+    // Verify every event after the first is the same object
+    if (events.length) {
+      assert.strictEqual(event, lastEvent);
```

Comment:
```suggestion
      assert.ok(event === lastEvent);
```
According to the comment it should be the same object not just equal content.